File: ApiSuvesaPos/Negocio/Modelo/caja/arqueocajas.vb
Namespace Modelo
    Public Class arqueocajas
        Public Property Id As Long
        Public Property EfectivoColones As Decimal
        Public Property EfectivoDolares As Decimal
        Public Property TarjetaColones As Decimal
        Public Property TarjetaDolares As Decimal
        Public Property Cheques As Decimal
        Public Property ChequesDol As Decimal
        Public Property DepositoCol As Decimal
        Public Property DepositoDol As Decimal
        Public Property Total As Decimal
        Public Property IdApertura As Integer
        Public Property Fecha As DateTime
        Public Property Cajero As String
        Public Property Anulado As Boolean
        Public Property TipoCambioD As Decimal
        Public Property Observaciones As String
        Public Property TarjetaSistema As Decimal
        Public Property OtrasTarjetas As Decimal
        Public Property ArqueoDeposito As New List(Of Modelo.arqueodeposito)
        Public Property ArqueoeEectivo As New List(Of Modelo.arqueoefectivo)
        Public Property ArqueoTarjeta As New List(Of Modelo.arqueotarjeta)
    End Class

    Public Class arqueodeposito
        Public Property Id As Integer
        Public Property IdArqueo As Integer
        Public Property Banco As String
        Public Property Cuenta As String
        Public Property Moneda As String
        Public Property Numero As String
        Public Property Monto As Decimal
        Public Property IdApertura As Integer
        Public Property Tipo As String
    End Class

    Public Class arqueoefectivo
        Public Property Id As Integer
        Public Property Id_Arqueo As Long
        Public Property Id_Denominacion As Long
        Public Property Monto As Decimal
        Public Property Cantidad As Integer
    End Class

    Public Class arqueotarjeta
        Public Property Id As Integer
        Public Property Id_Arqueo As Long
        Public Property Id_Tarjeta As Integer
        Public Property Monto As Decimal
    End Class
End Namespace

File: ApiSuvesaPos/ApiSuvesaPos/Modelo/caja/arqueocajas.cs
using System;
using System.Collections.Generic;

namespace Modelo
{
    public class arqueocajas
    {
        public long Id { get; set; }
        public decimal EfectivoColones { get; set; }
        public decimal EfectivoDolares { get; set; }
        public decimal TarjetaColones { get; set; }
        public decimal TarjetaDolares { get; set; }
        public decimal Cheques { get; set; }
        public decimal ChequesDol { get; set; }
        public decimal DepositoCol { get; set; }
        public decimal DepositoDol { get; set; }
        public decimal Total { get; set; }
        public int IdApertura { get; set; }
        public DateTime Fecha { get; set; }
        public string Cajero { get; set; }
        public bool Anulado { get; set; }
        public decimal TipoCambioD { get; set; }
        public string Observaciones { get; set; }
        public decimal TarjetaSistema { get; set; }
        public decimal OtrasTarjetas { get; set; }
        public List<Modelo.arqueodeposito> ArqueoDeposito { get; set; } = new List<Modelo.arqueodeposito>();
        public List<Modelo.arqueoefectivo> ArqueoeEectivo { get; set; } = new List<Modelo.arqueoefectivo>();
        public List<Modelo.arqueotarjeta> ArqueoTarjeta { get; set; } = new List<Modelo.arqueotarjeta>();
    }

    public class arqueodeposito
    {
        public int Id { get; set; }
        public int IdArqueo { get; set; }
        public string Banco { get; set; }
        public string Cuenta { get; set; }
        public string Moneda { get; set; }
        public string Numero { get; set; }
        public decimal Monto { get; set; }
        public int IdApertura { get; set; }
        public string Tipo { get; set; }
    }

    public class arqueoefectivo
    {
        public int Id { get; set; }
        public long Id_Arqueo { get; set; }
        public long Id_Denominacion { get; set; }
        public decimal Monto { get; set; }
        public int Cantidad { get; set; }
    }

    public class arqueotarjeta
    {
        public int Id { get; set; }
        public long Id_Arqueo { get; set; }
        public int Id_Tarjeta { get; set; }
        public decimal Monto { get; set; }
    }
}
